src/main/resources/Sessions/Sessions.drl

```diff
@@ -38,11 +38,44 @@ rule "Session does not have a judge 4 weeks or less before start"
         Year($year: value)
         Month($month: value)
         Day($day: value)
-        $s : Session(judgeId == null && DateTimeUtils.between(start, $year, $month, $day, 0, 28))
+        $s : Session(judgeId == null && DateTimeUtils.between(start, $year, $month, $day, 15, 29))
     then
         insertLogical(new Problem(ProblemTypes.Resource_missing,
                                   ProblemSeverities.Warning,
-                                  "Session does not have a judge 4 weeks or less before start",
+                                  String.format("Session of type %s on %s does not yet have a judge assigned",
```

Comment:
I need to have a closer look at this

Reply:
something wrong with provided text?

Reply:
nope, see below, a different appraoch to the solution